windows-nsis/openvpn.nsi

```diff
@@ -428,6 +428,11 @@ Function .onInit
 	${GetParameters} $R0
 	ClearErrors
 
+${IfNot} ${AtLeastWinVista}
+	MessageBox MB_OK "Windows Vista and above required"
+	Quit
+${EndIf}
+
```

Comment:
Spaces instead of tabs (or vice versa) here? Otherwise looks like it should work (have not tested yet).